components/message/theme.css

```diff
@@ -7,6 +7,15 @@
   align-items: center;
 }
 
+.content {
+  flex: 2;
+}
+
+.image {
+  flex: 1;
+  text-align: center;
```

Comment:
PP: `text-align: center;` for an svg? Just flex this and `justify-content: center`

Reply:
You are right @InstaK, we can use flexbox props here.